api/src/main/kotlin/io/spine/protodata/type/TypeSystem.kt

```diff
@@ -185,3 +185,30 @@ public fun TypeSystem.resolve(fieldPath: FieldPath, message: MessageType): Field
     val nextMessage = nextMessageInfo.first
     return resolve(remainingPath, nextMessage)
 }
+
+/**
+ * Obtains a message type by its name.
+ *
+ * @throws IllegalArgumentException if the requested type is unknown for this [TypeSystem].
```

Comment:
All extensions throw more or less the same exception, but for some reason it is only documented here. Please adjust other descriptions accordingly.